example/package.json

```
@@ -6,26 +6,28 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
+    "lint": "next lint",
+    "sanity:graphql": "sanity graphql deploy",
+    "reset-mux-credentials": "sanity documents delete secrets.mux"
   },
   "dependencies": {
-    "@mux/mux-player-react": "^2.3.3",
-    "@portabletext/react": "^3.0.11",
-    "@sanity/client": "^6.14.0",
+    "@mux/mux-player-react": "^2.6.0",
+    "@portabletext/react": "^3.0.18",
+    "@sanity/client": "^6.18.2",
     "@sanity/image-url": "^1.0.2",
-    "@sanity/vision": "^3.30.0",
-    "next": "^14.1.0",
-    "next-sanity": "^8.0.0",
-    "react": "^18.2.0",
-    "react-dom": "^18.2.0",
-    "sanity": "^3.30.0",
-    "sanity-plugin-mux-input": "^2.3.0",
-    "styled-components": "^6.1.8"
+    "@sanity/vision": "^3.42.1",
+    "next": "^14.2.3",
+    "next-sanity": "^9.0.20",
+    "react": "^18.3.1",
+    "react-dom": "^18.3.1",
+    "sanity": "^3.42.1",
+    "sanity-plugin-mux-input": "file:.yalc/sanity-plugin-mux-input",
```

Looks like some `@sanity/plugin-kit` stuff used for dev is left behind 🙌 
```suggestion
    "sanity-plugin-mux-input": "^2.3.0",
```

---

@stipsan sorry for the wait and thank you for the review!

I'm not sure how to proceed here, because if we go with 2.3.0 it'll be the old version of the plugin. Shouldn't the example include the upcoming `2.4.0`?

In any case, feel free to go with this suggestion if I'm mixing things up, I was never fully certain of how the demo works for this repo 🙈